NSRetailPOS/NSRetailPOS/NSRetailPOS/UI/frmPaymentStatus.cs

```diff
@@ -0,0 +1,121 @@
+﻿using DevExpress.XtraEditors;
```

Comment:
I think the status check should be on a background thread, which keeps hitting until there is a success response, checking for status in button click eats up user attention. Only the cancel button should be enabled